lib/std/sha1.nim

```diff
@@ -7,40 +7,38 @@
 #    distribution, for details about the copyright.
 #
 
-## **Note:** Import ``std/sha1`` to use this module
```

Comment:
Why should this be removed?

Reply:
It should be obvious, since it's linked as `std/sha1` and the other `std/` modules don't include this note either. But I can put the note back, if you prefer, though a better solution IMO would be to make `nim doc` generate `std/sha1` as heading.

Reply:
> a better solution IMO would be to make nim doc generate std/sha1 as heading.

Agreed. But until that becomes the reality, we should leave the original note in.

Reply:
=> https://github.com/nim-lang/Nim/pull/16999